prow/plugins.yaml

```diff
@@ -184,21 +29,13 @@ plugins:
   - owners-label
   - size
   - trigger 
-  GoogleCloudPlatform/gke-vault-demo:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger
+
   GoogleCloudPlatform/oss-test-infra:
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger
+  - blunderbuss
+  - config-updater
+  - golint
+  - heart
+  - pony
+  - shrug
+  - wip
+  - yuks
```

Comment:
It is unclear why most of these are only enabled for oss-test-infra. In particular `wip`, `blunderbuss`, and `golint`.

Reply:
cc @chrislovecnm 

we probably don't want to change existing behavior for other repos now... assume they choose this list of plugins for a reason?